src/js/view/floating-drag-ui.js

```diff
@@ -1,5 +1,8 @@
 import UI from 'utils/ui';
 import { style } from 'utils/css';
+import { OS } from 'environment/environment';
+
+const _isMobile = OS.mobile;
```

Comment:
Don't make this a const. Use `OS.mobile` inline. It's a getter so it's better to call it later than up here where you slow down the eval of controls script.

Reply:
@pajong @DanFerrer ^^

Reply:
Please see what I wrote in the PR message - it prevents click on desktop

Reply:
@pajong that is an issue with UI.js, not with setting `preventScrolling`. As I pointed out, you want to prevent scrolling on all touch devices, not just mobile. We should either update the pointer event handling or use a different element for dragging.